app/src/main/res/layout/fragment_derivable_accounts.xml

```diff
@@ -51,25 +59,33 @@
     <TextView
         android:id="@+id/titleTextView"
         style="@style/UiKit.TextAppearance.Regular.Text3"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_margin="16dp"
         android:background="@drawable/bg_cloud_rounded"
         android:padding="16dp"
         android:text="@string/derivation_path_info"
-        android:textColor="@color/textIconPrimary" />
+        android:textColor="@color/textIconPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/derivationPathInputLayout" />
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/accountsRecyclerView"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_weight="1"
+        android:clipToPadding="false"
+        android:paddingBottom="96dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/titleTextView"
         tools:listitem="@layout/item_token_simple" />
 
     <org.p2p.uikit.components.UiKitButton
         android:id="@+id/restoreButton"
```

Comment:
can you rename this ids to match the format
buttonRestore
recyclerViewAccounts
textViewTitle 
etc
?

Reply:
sure, no problem